astronomer/providers/amazon/aws/xcom_backends/s3.py

```diff
@@ -0,0 +1,67 @@
+import json
+import os
+import uuid
+from typing import Any, Union
+
+import pandas as pd
+from airflow.models.xcom import BaseXCom
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+
+
+class S3XComBackend(BaseXCom):
+    """
+    Custom XCom persistence class extends base to support various datatypes.
+    To use this XCom Backend, add the environment variable `AIRFLOW__CORE__XCOM_BACKEND`
+    to your environment and set it to
+    `astronomer.providers.amazon.aws.xcom_backends.s3.S3XComBackend`
+    """
+
+    PREFIX = os.getenv("PREFIX", "S3XCOM_")
+    AWS_CONN_ID = os.getenv("CONNECTION_NAME", "aws_default")
+    BUCKET_NAME = os.getenv("XCOM_BACKEND_BUCKET_NAME", "some_bucket_name")
```

Comment:
do we also need a folder/file path inside a bucket or does BUCKET_NAME contain a full path?

Reply:
can we change the default bucket name to some cleaner name like `airflow_xcom_backend_default_backet`

Reply:
BUCKET PATH will take care of it.